.github/workflows/tests.yml

```diff
@@ -237,7 +237,7 @@ jobs:
     needs:
       - source
       - build_macos_arm64
-    runs-on: macos-14
+    runs-on: macos-14-arm64
```

Comment:
@pimpin does it work? according to official page there is no such runner:

https://github.com/actions/runner-images/blob/main/README.md#available-images

Reply:
Indeed it's not listed in the README, but it is listed in MacOs images : https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md